packages/forma-36-website/src/content/migration-v3-to-v4.mdx

```diff
@@ -34,6 +34,10 @@ To migrate your project easily, we provided you with a codemod for most of the c
 
 ✅ IconButton: Now you can pass any icon as an element to IconButton component. For more information on how to migrate your IconButton components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-iconbutton-components).
 
+✅ DateTime: You only need to update `format` prop value. For more information on how to migrate your DateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
+
+✅ RelativeDateTime: No changes, just import new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```

Comment:
```suggestion
✅ RelativeDateTime: No changes, just import the new component. For more information on how to migrate your RelativeDateTime components, see [the detailed documentation](https://github.com/contentful/forma-36/blob/forma-v4/MIGRATION.md#how-to-migrate-your-datetime-components)
```

Just a nitpicky thing, I think we need a "the" over there
But I'm not native speaker or good with grammar 👀